### DISCUSSION TITLE
Nova2 PtBR doesn't understand 'one'

### DISCUSSION THREAD

**Discussion by Shoddyteam** [deepgram/community#488] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Nova2 on the language brazillian portuguese doesn't understand 'one'. More specifically, if you say 'um' (one) it just skips the word. Looking at the JSON, it's almost as if the word didn't even exist at all. For context, the transcriptions are supposed to be for measuring, so for instance 1,5 x 1,1 cm (it would read that as' ,5 x , cm'). 

### If possible, please attach an example audio file to reproduce the issue.

[3efa3636-6696-463f-b27d-bb6a5d35cfef.zip](https://github.com/deepgram/community/files/13620853/3efa3636-6696-463f-b27d-bb6a5d35cfef.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@Shoddyteam  I'd recommend you give Nova-2 another shot at handling this audio now. Our Models are ever-evolving with auto-training so they do get more accurate over time. If you still see problem here just let us know.
